chirho/dynamical/internals/_utils.py

```diff
@@ -48,8 +48,8 @@ def append(fst, rest: T) -> T:
     raise NotImplementedError(f"append not implemented for type {type(fst)}.")
 
 
-@append.register(Trajectory)
-def _append_trajectory(traj1: Trajectory[T], traj2: Trajectory[T]) -> Trajectory[T]:
```

Comment:
Could we rename this to `_append_state` and call the arguments `state1` and `state2`?

Reply:
This function is never used except through `append`, so I don't think it matters. I hope to be able to delete `append` entirely, but it will require more work addressing #328 and other related issues.